b2luigi/batch/processes/lsf.py

```diff
@@ -73,10 +79,12 @@ def get_job_status(self):
     def start_job(self):
         command = ["bsub", "-env all"]
 
-        try:
-            command += ["-q", self.task.queue]
-        except AttributeError:
-            pass
+        queue = get_setting("queue", task=self.task, default='s')
```

Comment:
Was ist intended to have a default here? While "s" might be the best default setting for the KEK batch system, that is not something "universal" (and e.g. people running on other batch farms will need to change that). I would love to make this part as general as possible.

Reply:
good point! I will fix that. I think for lsf it should still default to 's' anyway. I'm not familiar with the other batch systems, do they have a 'job_name' parameter or something analogous?

Reply:
Concerning your question, I just searched the documentation for `condor_submit` and it has a 
`-batch-name` setting which seems to serve an identical purpose to the lsf `job_name`. This could be handled by the same setting.

Gbasf2 projects have a project name, which for gbasf2 is required, since for gbas2 you usually submit a steering file and an input dataset and gbasf2 handles the creation of individual jobs for that dataset, grouped under that project name. The b2luigi gbasf2 wrapper that I wrote is therefore different from the htcondor or lsf BatchProcess'es, as it treats the whole project (group of jobs) as an individual job. In that case I had to make the project name a required setting and also make sure that it is unique, so I ask the users to provide a `gbasf2_project_name_prefix` setting and automatically add a unique hash as a suffix to the project name.